code/modules/mob/living/living_defense.dm

```diff
@@ -278,7 +279,7 @@
 		adjustFireLoss(SENTINEL_TOXIC_GRENADE_GAS_DAMAGE * bio_protection)
 	if(CHECK_BITFIELD(S.smoke_traits, SMOKE_CHEM))
 		S.reagents?.reaction(src, TOUCH, S.fraction)
-
+	return protections
```

Comment:
What's the reason for returning a full list when only bio gets used?

Reply:
Yup, It used to work with 1 var, and refactor added a few more. Its now just bio